src/plugins/flexibleLayout/utils/container.js

```diff
@@ -0,0 +1,14 @@
+import Frame from './frame';
+
+class Container {
+    constructor (width) {
```

Comment:
We want to avoid inserting classes inside domain objects-- this should be a factory function which returns a new javascript object.